src/components/report/TermsAndConditionsDialog.jsx

```diff
@@ -1,24 +1,786 @@
 import React from 'react';
 import DialogContent from '@material-ui/core/DialogContent';
-import Text from '../Text';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
 import StandardDialog from '../StandardDialog';
 
+function H1({ children, ...rest }) {
```

Comment:
I wonder whether a lot of these could be put in some kind of component utils file somewhere so that they're move easily accessible for other components as well.

@brmscheiner , thoughts?

Reply:
yeah, this work could definitely be helpful for building some other text-heavy pages, plus this file is already super long. i think we should move these component definitions to the `src/components` directory. also i don't think we should name components the same thing as html elements, i feel like that could create confusion. maybe Header1 or something?

Reply:
Changed the names (https://github.com/WildMeOrg/codex-frontend/pull/275/commits/b1ed9bffd8c1f4ca8dab552d0b1c3d6cc81d9bac) and moved to a separate file (https://github.com/WildMeOrg/codex-frontend/pull/275/commits/1df0aabab58cae57d74e9ac3603a465a54de2011).